website/src/pages/index.js

```diff
@@ -106,6 +106,8 @@ function Home() {
               Get Started
             </Link>
           </div>
+          <p className="hero__subtitle"></p>
```

Comment:
What is this for?

Reply:
This is to add spacing between Get Started button and GH star button